### PR TITLE
сmd: disable flag

### LINKED ISSUE #294: cmd/criticize: add `-disable` flag that turns off specified checks
Now, if user needs to disable one checker he should enable all besides one.

### PR DESCRIPTION
**Pull request by cristaloleg**
Fixes #294

### CODE REVIEW

File: cmd/criticize/main.go [hunk 1]
```diff
@@ -98,9 +101,19 @@ func parseArgv(l *linter) {
 		blame("-withOpinionated used with -enable=%q", *enable)
 	}
 
+	fmt.Printf("`%v` `%v`\n", *enable, *disable)
```
Comment (cristaloleg) [inline cmd/criticize/main.go]: ouch

File: cmd/criticize/main.go [hunk 2]
```diff
@@ -98,9 +101,19 @@ func parseArgv(l *linter) {
...
+		for _, rule := range lint.RuleList() {
+			if rule.Experimental && !l.withExperimental {
+				continue
+			}
+			if rule.VeryOpinionated && !l.withOpinionated {
+				continue
+			}
+			l.enabledCheckers = append(l.enabledCheckers, rule.Name())
+		}
 	case "":
```
Comment (cristaloleg) [inline cmd/criticize/main.go]: Do we need this after introducing `-disable` ?

**Comment by Quasilyte** [comment]
Do we need `-withExperimental` and `-withOpinionated`?

File: cmd/criticize/main.go (hunk 2 above)
Comment (Quasilyte) [reply cmd/criticize/main.go]: I guess not.

**Comment by Quasilyte** [comment]
I'll review this after #340.

**Comment by cristaloleg** [comment]
I think we should left `-withExperimental` and `-withOpinionated. They're quite descriptive and usable.

**Pull request merged**